qcengine/programs/psi4.py

```diff
@@ -53,36 +60,39 @@ def compute(self, input_model: 'ResultInput', config: 'JobConfig') -> 'Result':
         Runs Psi4 in API mode
         """
         self.found(raise_error=True)
-        import psi4
 
         # Setup the job
-        input_model = input_model.copy().dict()
-        input_model["nthreads"] = config.ncores
-        input_model["memory"] = int(config.memory * 1024 * 1024 * 1024 * 0.95)  # Memory in bytes
-        input_model["success"] = False
-        input_model["return_output"] = True
+        input_data = input_model.json_dict()
+        input_data["nthreads"] = config.ncores
+        input_data["memory"] = int(config.memory * 1024 * 1024 * 1024 * 0.95)  # Memory in bytes
+        input_data["success"] = False
+        input_data["return_output"] = True
 
-        if input_model["schema_name"] == "qcschema_input":
-            input_model["schema_name"] = "qc_schema_input"
+        if input_data["schema_name"] == "qcschema_input":
+            input_data["schema_name"] = "qc_schema_input"
```

Comment:
as of 1.3, psi will take either (changes to qc_ for qcel 0.1.3's sake)

Reply:
Yea, but doesn't hurt anything at the moment and is still backward compatible.